Makefile

```diff
@@ -1,8 +1,9 @@
-BUILD_VARIANT=release
-DD_TRACE_GO_VERSION:=v1.34.0
+BUILD_VARIANT       ?= release
+DD_TRACE_GO_VERSION ?= v1.36.0
+BUILD_TAGS          := $(BUILD_VARIANT),$(BUILD_TAGS)
```

Comment:
does it make sense to have both `BUILD_VARIANT` and `BUILD_TAGS`? Or wouldn't it be simpler to just have `BUILD_TAGS`?

Reply:
Sure yes. I felt like you wanted "business-specific" parameters so I added a separate BUILD_TAGS one, but all good to merge both into a single one.

Reply:
My bad @felixge: it's actually not possible due to the following `-X main.version=$(BUILD_VARIANT)/git describe --tags HEAD` that would be impossible to deduce out of `BUILD_TAGS` alone.

WDYT? Do I merge the way it is?

Reply:
Yeah, let's just merge it. We can change it again anytime.